tests/app/service/test_rest.py

```diff
@@ -941,12 +941,12 @@ def test_add_unknown_user_to_service_returns404(notify_api, notify_db, notify_db
             assert result['message'] == expected_message
 
 
-def test_remove_user_from_service(notify_api, notify_db, notify_db_session, sample_service_permission):
+def test_remove_user_from_service(notify_api, notify_db, notify_db_session, sample_user_service_permission):
```

Comment:
You can also use the `client` fixture here to get rid of all the `with` statements.